tasks/enable.yml

```
@@ -0,0 +1,111 @@
+---
+- name: Check trust compatibility
+  fail:
+    msg: >-
+      Fapolicyd does not support trust setting fapolicyd_setup_trust
+      on EL version < 8.3
+  ignore_errors: true
+  when:
+    - fapolicyd_setup_trust is not none
+    - ansible_facts.distribution_version is version("8.2", "<=")
+  register: __failed_check_trust
+
+- name: Check integrity compatibility
+  fail:
+    msg: >-
+      Fapolicyd does not support integrity setting fapolicyd_setup_integrity
+      on EL version < 8.4
+  ignore_errors: true
+  when:
+    - fapolicyd_setup_integrity is not none
+    - ansible_facts.distribution_version is version("8.3", "<=")
+  register: __failed_check_integrity
+
+- name: Check trust files compatibility
+  fail:
+    msg: >-
+      Fapolicyd does not support trust files setting fapolicyd_add_trusted_file
+      on EL version < 8.4
+  ignore_errors: true
+  when:
+    - fapolicyd_add_trusted_file is not none
+    - ansible_facts.distribution_version is version("8.3", "<=")
+  register: __failed_check_trusted_file
+
+- name: Check failed conditions
+  fail:
+    msg: Multiple failed conditions
+  when: __failed_check_trust is failed or __failed_check_integrity is failed  or
+    __failed_check_trusted_file is failed
+
+- name: Install fapolicyd packages
+  package:
+    name:
+      - "{{ __fapolicyd_packages }}"
+    state: present
+
+- name: Install fapolicyd-selinux packages
+  package:
+    name:
+      - "{{ __fapolicyd_selinux_packages }}"
+    state: present
+  when: ansible_facts.distribution_version is version("8.3", ">=")
+
+- name: Copy fapolicyd configuration file
+  template:
+    src: "{{ __fapolicyd_conf }}.j2"
+    dest: "{{ __fapolicyd_dir }}/{{ __fapolicyd_conf }}"
+    owner: root
+    group: fapolicyd
+    mode: '0644'
+
+- name: Run fapolicyd configuration check
+  command: fapolicyd-cli --check-config
+  check_mode: false
+  changed_when: false
+  when: ansible_facts.distribution_version is version("8.6", ">=")
+
+- name: Trustdb cleanup
+  command: fapolicyd-cli --file delete /
+  when: fapolicyd_add_trusted_file is not none
+  changed_when: true
+  failed_when: false
+
+- name: Add file to trustdb
+  command: fapolicyd-cli --file add "{{ item | quote }}"
```

```suggestion
  command: fapolicyd-cli --file add {{ item | quote }}
```
using the `| quote` filter already does the necessary quoting/escaping to the value for the shell - you don't need to use additional double quotes